pkg/manager/internal.go

```diff
@@ -355,6 +359,19 @@ func (cm *controllerManager) serveHealthProbes() {
 	go cm.httpServe("health probe", cm.logger, server, cm.healthProbeListener)
 }
 
+func (cm *controllerManager) servePprof() {
+	mux := http.NewServeMux()
+	server := httpserver.New(mux)
+
+	mux.HandleFunc("/debug/pprof/", pprof.Index)
+	mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
```

Comment:
This is really interesting and helpful. However, if we support this, how we allow users to expose custom profiles?

Reply:
Users can access their custom profiles through `/debug/pprof/*` which is already exposed by `mux.HandleFunc("/debug/pprof/", pprof.Index)`.